docs/config/index.md

```diff
@@ -24,14 +24,14 @@ vite --config my-config.js
 ```
 
 ::: tip NOTE
-Vite will replace `__filename`, `__dirname`, and `import.meta.url` in config files and its deps. Using these as variable names or passing as a parameter to a function with string double quote (example `console.log`) will result in an error:
+Vite will inject `__filename`, `__dirname` in config files and its deps. Declaring these variables at top level will result in an error:
```

Comment:
In CJS, the following code fails, so it is not a special thing.
```js
const __dirname = ''
```
But here it happens with ESM. This is the reason I left this note.

Also I didn't mention `import.meta.url` because `__vite_injected_original_import_meta_url` will rarely conflict.